nestkernel/nest_time.h

```diff
@@ -387,7 +387,13 @@ class Time
   bool
   is_neg_inf() const
   {
-    return tics == LIM_NEG_INF.tics;
+    return tics <= LIM_NEG_INF.tics;
```

Comment:
@apeyser As far as I can see from the `nest_time.h` code, `tics` can never become smaller than `LIM_NEG_INF.tics`; the only way for this to happen would be a `Time` method calling the protected `Time(tic_t)` constructor. I still had suggested the change above from the perspective of defensive programming: if `LIM_NEG_INF.tics` represent negative infinity, any smaller value cannot be larger and thus must be infinity as well.

On the other hand, one could argue that `LIM_NEG_INF.tics` is a "magic value" (just as Inf for doubles is represented by a specific bit pattern), and that we should match that exactly.